wagtail/admin/views/generic/models.py

```diff
@@ -403,12 +403,14 @@ def get_success_url(self):
     def get_success_message(self, instance):
         if isinstance(instance, DraftStateMixin) and self.action == "publish":
             if instance.go_live_at and instance.go_live_at > timezone.now():
-                return _("'{0}' created and scheduled for publishing.").format(instance)
-            return _("'{0}' created and published.").format(instance)
+                return _("'%(object)s' created and scheduled for publishing.") % {
```

Comment:
Since we're replacing these strings anyway, could you please also edit the strings in the snippets views to use more generic variable names?

Specifically, I need all occurrences of `snippet_type` to be renamed to something else e.g. `model_name`:
 https://github.com/wagtail/wagtail/blob/ef8a9702614448edaf1ee66bbbd13b42fc004dab/wagtail/snippets/views/snippets.py#L245-L257

Then, it's up to you to use `object` or `instance`, but make sure to be consistent for both views.

Once we've renamed the snippet-specific variables, we can use those strings in the generic views (they're more informative as they also provide the model name).

I think `snippet_type` is the only variable in the snippets views that is snippet-specific, but if you found any other ones please also rename them to be more generic. Thanks!

Reply:
Maybe we can do this rename as a follow up PR? Might be good to land this in `main` soon. This way new PRs can come in with the linting rules.

Reply:
I'll merge in the next hour or two. If there aren't too many changes for snippets, I'll try to do it at the same time. Otherwise, I'll leave it to a follow-up PR indeed.

Reply:
Done in 6d9476d with some additional normalization in 868c239.
Corresponding [diffs update](https://gist.github.com/loicteixeira/bcc5e678e3d06af07bdb3afe9d149575/revisions#diff-b6e4f31103721ae34912b0ec3248c72d9054c3db48c3e40b97d8129cb328d8fd) and [script update](https://gist.github.com/loicteixeira/bcc5e678e3d06af07bdb3afe9d149575/revisions#diff-c639433905560e28f8d413989b641d1b2b4773333faa845142da87b8a08275eb).

Onto the merge and Transifex update now 🤞

Reply:
Thanks Loic. You are a legend